### ISSUE TITLE
Vision document principles and values mixes values for the web with values for the consortium

### ISSUE THREAD

**Issue by dwsinger** [w3c/AB-public#36]
We need to explain to the outside world what the values we hold for the web are, without entangling the values we have for how we treat our members. The current document mixes them. This is confusing.

These are principles for the web

> Ensure the Web is trustworthy, by ensuring security and privacy for users.
> Ensure the Web does not favor centralization.
> Implement a unified, extensible, Web architecture, which continues to address evolving use cases for the general public.

These are principles for the consortium

> Remain focused on interoperability and collective empowerment, and ensure our work is supported by open test suites.
> Encourage incubation in new areas and industries with open platforms for discussion, collaboration and innovation, making it more structured and improving consensus-building among key stake holders.
> Strive for the broadest participation, along axes including worldwide participation, diversity and inclusion, facilitating balance, equity and cooperation among the participants from different industries, user groups and organizational sizes, and thus establishing W3C as representative of the whole community.
> Increase involvement of under-represented key stakeholders such as end users, content creators, developers etc.
> Ensure transparency, equity and fairness. Our work will not be exclusively dominated by any person, company or interest group.
> Establish and improve collaborative relationships with other organizations in the domain of Internet and Web standards, including building and maintaining respected relationships with governments and businesses for providing credible advice.

**Comment by cwilso** [comment]
We should take each of these points in tern.

1. Ensure the Web is trustworthy, by ensuring security and privacy for users.

The point of having this here is to capture that the W3C must take it as an explicit principle that it is a core function of the W3C to ensure user privacy and security.  We could of course express this better - I understand why you say it reads like a web principle rather than a W3C one, but it really is intended to capture a subtly different point.  

2. Ensure the Web does not favor centralization.

I'll agree this one reads much more as a web principle, not a W3C one.  I'd suggest perhaps it could simply move to the vision of the WWW section after "one interoperable" - but I would welcome explicit suggestions of what to do here.  I am particularly sensitive to personally filing changes on this without consensus.

3. Implement a unified, extensible, Web architecture, which continues to address evolving use cases for the general public.

I'll be direct - I wanted to simply drop this in the refactor, and decided not to as it was not just moving things around.  I think this "principle" is not a principle at all, and should simply be removed, and nothing important is lost by doing so.  But I was going to file a separate issue for that.

Thoughts?

**Comment by dwsinger** [comment]
I agree point 3 of the web principles states the obvious of what we do. But my point is that these principles that the web is built on, and the principles by which w3c operates, are two separate sets of principles and should be separated.

I also think we probably have one or two more principles that the web is built on. 2-3 seems rather few.

**Comment by cwilso** [comment]
Your point seems to be that we should have a section capturing "Principles of the Web" [as opposed to the Principles section we have, which is clearly intending to be principles for the W3C].  I think this is the space the TAG already fully delineates with the Ethical Web Principles (https://www.w3.org/TR/ethical-web-principles/), and I would hesitate to replicate that list of principles or change it; if we want to have more input into it we should work with the TAG to do so, but we should not duplicate work.  (This Vision document does reference the EWP; it could do so more directly for this purpose if that's deemed necessary.)

**Comment by dwsinger** [comment]
No, gosh, I thought my point was clear. I don't want to subsume the excellent but much longer Ethical Web Principles. I don't want to change the text here at all. I do want to separate the _outward_ principles, what we are doing for the web and hence for the world, from the _inward_ principles, what we do with and for our members.

Why? Because when we are talking to prospective members, or trying to attract them, we need both and they can be entwined (but I still think it better not).

But when we are talking to external bodies about why we matter, what we are doing for humanity and the world, it's those outward principles they will be focusing on. Yes, they may aks "do you treat your members well?" but they may not.

It's also good to see that we have a reasonably decent set of each.

So, in summary, I'm asking for the document organization to improve.

**Comment by cwilso** [comment]
I think you are now saying you are not looking for this document to contain principles that the web is [should be] built on - that is the realm of the TAG's EWP - but that we should separate the principles of what the Consortium should work on into external-facing (what we are doing for the web, e.g. "ensure privacy") and internal-facing (what we are doing for/with our members, e.g. "Increase involvement of under-represented key stakeholders".  Is that accurate?

**Comment by dwsinger** [comment]
Have a look at the [prior version](https://github.com/w3c/AB-public/tree/367217932c590a029adcb236b3158e2daa37ec83/Vision) of the document. It had two sections of principles: those that apply to what we build (the web itself) and "outside customers" (our action as a consortium), and those that apply to how we operate (our operating principles). This distinction was not an accident, it was deliberate. I'm simply asking for it to be restored. The audiences for these two are different, and if we're looking for support from non-members, it's the former base principles that apply most.

**Comment by cwilso** [comment]
"Restoring" this would 1) be in direct conflict with the goals of the restructure (i.e. I think it's the opposite of the simplification @avneeshsingh was looking for, and 2) would again confuse two lists that appear to be quite similar - they are, in fact, both things that the Consortium should do/operating principles the Consortium should act under.

If your point is truly just "we should be identifying the categories of 'things that are external-focused' vs 'things that are internal-focused'," I can get behind that and see how to take a stab at it.  If so, this issue is really "Clearly delineate internal- and external-focused principles".

If your point is "we should list out the principles the web is based on" (you did say that, in the first line of this issue), then I think we should be leaving that largely to the TAG EWP.

If your point is "restore what was there, it was perfect the way it was", then I think there is substantial disagreement with that structure, and you are welcome to have a go at an edit.  And you probably wanted to state opposition to https://github.com/w3c/AB-public/pull/30, but I guess it's a bit late now.  But just restoring is counter to a large part of the goal of the restructure that was done in #30.

If I'm missing your point entirely, please try explaining again, and my apologies for being obtuse.  I am really not trying to be.

**Comment by dwsinger** [comment]
Overall, #30 was fine. But this is one of two aspects (the title was the other) where the document is now more _complex_ than it was. 

If those two sections need better titles, I could agree with that. The TAG EWP is where we got many of these base principles, and it develops them in a consistent way. We should absolutely have both a short list of core principles here, and the working-through that the EWP represents. They should both exist, and be consistent.

I started the Vision because I thought we'd be looking for external support, and to get that we'd need to explain to the world why we matter to the world: what are we doing that positively impacts humanity. That's still, for me, the top priority. But it's morphing into a completely different document that explains our internal operating principles. That's important too, but I feel the original motivation has been hijacked.

Frankly, I'd prefer that they were separate documents. But since they should both be short, and they should be consistent with each other, and there may be some overlap, I'm OK with them being in one document, as long as the two different messages and foci can be distinguished. 

On the other hand, if the AB wants to make it a document about our internal operating principles, then I'll start a new external vision document somewhere else (probably at the Board, but I'd prefer it was more member-owned). At the moment, the document confusingly mixes what we do for the world with how we handle our members.

So yes, I'd be fine with reverting this one aspect of #30, and I'd be fine with better titles for the outward and inward facing principles sections. But if we are to go outside the W3C for support, I need a consensus document that I can extract that explains "what are we doing for humanity, what does the web mean for the world" without getting into how we're nice to our members. I agree that if you are a consortium member, they are both lists of principles we need to act under. That doesn't mean it makes sense to mix them up. That mixes product design criteria with team management criteria ("we're going to deliver a browser that is usable in as many of the world's scripts as possible" and "we're going to hire team members explicit looking for diversity in gender, country of origin, accessibility, etc.").

**Comment by TzviyaSiegman** [comment]
@dwsinger I think I understand what you are getting at. If I am understanding correctly, you would like to see a document that describes not only what the W3C has historically stood for but has never documented but also explains why W3C continues to change the world. This project began as an exploration about how W3C could help prevent unintended consequences in the future, but there is nothing explaining how W3C will do that. Is that what you'd like to see? 

I think that can definitely be added in a section called something like "Our Commitment to the Future Web".

**Comment by dwsinger** [comment]
So, the old document had, for our principles for the web itself:

> We will continue to develop a world-wide web that
> 
> - Works for everyone, striving for diversity and inclusion of participants from different geographical locations, cultures, languages, accessibility needs, gender identities, and more. We strongly emphasize accessibility, internationalization, privacy, security and diversity.
> - Puts the needs of users first: above authors, publishers, implementers, paying W3C Members, or theoretical purity.
> - Is trustworthy, by ensuring security and privacy for users.
> - Does not favor centralization.
> - Implements a unified, extensible, Web architecture, which continues to address evolving use cases for the general public.
> - Is based on standards that are openly developed with consensus of industry and key stakeholders, standards rooted in a strong royalty-free patent policy and open copyright licenses, focussed on interoperability and collective empowerment, and supported by open test suites.

The first bullet seems to have gone completely (I'd be fine with losing the second sentence, which combines re-stating the first with some other unrelated points, but the first sentence is important). The next 4 bullets are still there but mixed into a general principles section. The last is a cross-over; it's important both for the world (open licenses and RF patent policy make the technologies adoptable and adaptable) but is also a statement of how we work. It's in the "vision for the w3c" now and it's OK to leave it there.

So, trying again, can we extract the latest edited versions of bullets 2-5, put back a suitable edited version of bullet 1, and make it a section of its own, or add it to the Vision for the World-Wide Web section?

**Comment by cwilso** [comment]
The first bullet is NOT gone entirely: the detailed commits Tantek demanded clearly show what happened (https://github.com/w3c/AB-public/pull/30).  I would suggest reviewing that point by point if you really want to understand how things moved around.  The initial bullet broke into three points:

- Works for everyone [this was removed, as it's redundant imo with "the web is for all humanity" , "we put the needs of users above...[list]", et al.    You may disagree with this; I'm still not sure what point it's making on top of the others, though, and this document is too long, confusing and repetitive.  But if the sticking point is that we don't have a point that says "the web is for everyone", I'm not gonna die on that hill.
- This organization believes in diversity and inclusion of participants from different geographical locations, cultures, languages, accessibility needs, gender identities, and more.  [ this is nearly identical to the second clause of that first bullet]
- We strongly emphasize accessibility, internationalization, privacy, and security. [This IS identical to the second sentence of the first bullet]

Rather that trying to "put back" and "revert edits", can we simply define what it is we're trying to have here?  What we had was NOT well organized and identified.  You seem to simply be asking that we separate out "how we operate" from "what we're trying to do for the Web."  Great, I can take a stab at that.

**Comment by dwsinger** [comment]
> You seem to simply be asking that we separate out "how we operate" from "what we're trying to do for the Web."

Yes, that's exactly what I am asking. I'm explicitly not asking to revert other textual changes.

I'm sorry I was unable to work out where the first bullet went; I searched for text and did try to find the appropriate commit (and failed). Apologies, but I did try.

> This organization believes in diversity and inclusion of participants from different geographical locations, cultures, languages, accessibility needs, gender identities, and more. [ this is nearly identical to the second clause of that first bullet]

Yes, but this is about _participants in W3C_ whereas the phrase that was removed was about the applicability of the _web itself_.

Similarly the statement about accessibility is now part of a section labeled "vision for the W3C", i.e. not apparently a vision for the web itself.

**Comment by cwilso** [comment]
The previous line was:
>We will continue to develop a world-wide web that
* Works for everyone, striving for diversity and inclusion of participants from different geographical locations, cultures, languages, accessibility needs, gender identities, and more. We strongly emphasize accessibility, internationalization, privacy, security and diversity.

I think you must have been reading "participants" here as participants in the Web, rather than participants in W3C?  But this was even then under the heading "Our action as a consortium", so I think it read as "participants in the W3C"; I don't think this was ever about the web itself, it was about the consortium (based on the section heading).

I think this all is covered by "the web is for everyone", but perhaps we should just expand there then.
